common/src/main/java/com/microsoft/identity/common/internal/platform/RawKeyAccessor.java

```diff
@@ -60,21 +61,40 @@
  * Key accessor for using a raw symmetric key.
  */
 @Builder
+@Getter
+@Accessors(prefix = "m")
 public class RawKeyAccessor implements KeyAccessor {
-    private final CryptoSuite suite;
-    private final byte[] key;
     private static final SecureRandom mRandom = new SecureRandom();
 
+    /**
+     * The cryptoSuite to use with this RawKeyAccessor.
+     */
+    @NonNull
+    private final CryptoSuite mSuite;
+    /**
+     * The byte array that backs this key.
+     */
+    @NonNull
+    private final byte[] mKey;
+
+    /**
+     * The alias for the stored key, may be null.
+     */
+    private final String mAlias;
```

Comment:
I don't see this ever being set

Reply:
Hm.  It wouldn't be, here.  I end up manufacturing these in the PRT code path, but I don't think we communicate a session key alias at that point.  I should set it in the random-key alias part of the KeyStoreAccessor, though.

Reply:
37ed32387b54c8540e6973e8c53ce52aa621eb72